Automattic/jetpack
# Admin Page: Downgrade to Node 6.11.5 and node-sass-3.7

This combination seems to work fine with most of the changes introduced by #7549.

This allows developers, designers and HEs to continue using a version that is compatible for building Calypso.

#### Changes proposed in this Pull Request:

* Downgrades travis.yml, dev docs, package.json, .nvmrc to using Node *6.11.5*
* Updates yarn.lock file to deal with the mentioned changes

#### Testing instructions:

* Test the following commands on the listed architectures and please check when you've tested
  ```
  $ yarn distclean && yarn clean-client && yarn cache clean && yarn build
  $ yarn distclean && yarn clean-client && yarn cache clean && yarn build-production
  ```
  - [x] MacOs latest Sierra
  - [x] MacOs High Sierra
  - [x] Linux-based box.
* Visit The Jetpack admin page. Confirm it's there.
